README.md

```diff
@@ -215,7 +215,7 @@ Optuna dashboard provides various visualizations to analyze the optimization res
 
 This demo showcases how to use the Optuna MCP server to automatically find optimal FFmpeg encoding parameters. It optimizes x264 encoding options to maximize video quality (measured by the SSIM score) while keeping encoding time reasonable.
 
-Check out [examples/ffmpeg.md](https://github.com/optuna/optuna-mcp/tree/main/examples/ffmpeg/ffmpeg.md) for details.
+Check out [examples/ffmpeg](https://github.com/optuna/optuna-mcp/tree/main/examples/ffmpeg/README.md) for details.
```

Comment:
I confirmed that the URL is correct: https://github.com/optuna/optuna-mcp/blob/main/examples/ffmpeg/README.md